package.json

```diff
@@ -48,16 +48,17 @@
     "babel-polyfill": "6.26.0",
     "intl": "1.2.5",
     "mixpanel-browser": "2.14.0",
-    "react": "16.1.1",
     "react-dom": "16.1.1",
-    "react-intl": "2.4.0",
     "react-intl-po": "2.1.1",
+    "react-intl": "2.4.0",
     "react-redux": "5.0.6",
-    "react-router": "4.2.0",
     "react-router-dom": "4.2.2",
-    "redux": "3.7.2",
+    "react-router": "4.2.0",
+    "react": "16.1.1",
```

Comment:
I guess we don't really agree on the alphabet here.

Reply:
Will post an issue on VS Code.. `-` < `"`